src/Oldolf/MarkerFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Oldolf;
+
+use AppBundle\Oldolf\Marker\MaisonServiceAccueilPublic;
+
+class MarkerFactory
```

Comment:
A factory should create something, here it's rather `MakerChoiceLoader`, WDYT?

Reply:
very shortly it will do the same job as the MeasureFactory : creating markers & custom paylods (we will need to display popups with infos on markers on the map)